flow-typed/redux-mock-store_v1.2.x.js

```diff
@@ -5,12 +5,15 @@ declare module 'redux-mock-store' {
   */
 
   declare type mockStore = { <S, A>(state: S): mockStoreWithoutMiddleware<S, A>, ... };
-  declare type DispatchAPI<A> = (action: A) => A;
-  declare type Dispatch<A: $ReadOnly<{ type: string, ... }>> = DispatchAPI<A>;
+  declare interface Dispatch<S, A> {
```

Comment:
Mm, the `S` type argument is unnecessary (making a comment so I remember to address this in my next revision, along with code-review feedback).

Reply:
(and a code comment here saying that we've edited this bit would be good too)